packages/picasso.js/src/core/component/animations/adjust-tweened-nodes.js

```diff
@@ -0,0 +1,9 @@
+export default function adjustTweenedNodes({ tweenedNodes, formatter }) {
+  tweenedNodes.forEach((node) => {
+    const { type, data } = node;
+    if (type === 'text' && data && data.value && data.formatterKey) {
```

Comment:
`formatterKey` seem to suggest that only a "key" can be used. But in reality since this just gets passed to `chart.formatter` any of the following would also be valid:

```
* instance.formatter('nameOfMyFormatter'); // Fetch an existing formatter by name
   * instance.formatter({ formatter: 'nameOfMyFormatter' }); // Fetch an existing formatter by name
   * instance.formatter({ type: 'q' }); // Fetch an existing formatter by type
   * instance.formatter({
   *  formatter: 'd3',
   *  type: 'number',
   *  format: '1.0.%'
   * }); // Create a new formatter
  ```

I suggest you change it to just `formatter` to better match reality.

Reply:
good suggestion. I will do that. Thanks.